.github/workflows/build-push.yml

```
@@ -0,0 +1,103 @@
+name: Build-Push
+
+on:
+  pull_request:
+  push:
+    branches:
+      - main
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Load git submodules
+      run: git submodule sync && git submodule update --init
+
+    - name: Install hugo
+      run: |
+        wget https://github.com/gohugoio/hugo/releases/download/v0.113.0/hugo_extended_0.113.0_Linux-64bit.tar.gz \
+        && tar -xvf hugo_extended_0.113.0_Linux-64bit.tar.gz hugo \
+        && mv  hugo /usr/local/bin \
+        && rm hugo_extended_0.113.0_Linux-64bit.tar.gz
+    - name: Build english and french sites
```

nit: could you add a newline:
```suggestion

    - name: Build english and french sites
```